app/components/wds-global-navigation/link.js

```diff
@@ -3,5 +3,5 @@ import Component from '@ember/component';
 export default Component.extend({
 	tagName: 'a',
 	attributeBindings: ['model.href:href'],
-	classNames: ['wds-global-navigation__link']
+	classNameBindings:['standaloneLink:wds-global-navigation__link'],
```

Comment:
I wonder if this should be component's property or we should just add `class='wds-global-navigation__link'` in template invocation where we need it. @hakubo @latata what do you think?